config/esp32/components/chip/Kconfig

```diff
@@ -755,6 +755,16 @@ menu "CHIP Device Layer"
                 When size is set to 0, the debug event buffer and all support
                 for the debug level events are disabled.
 
+
+	config CHIP_ENABLE_SCHEMA_CHECK
+	   bool "Enable Schema Check"
+	   default n
+	   help
+	     If enabled, it checks incoming messages are following the expected schema,
+             and incoming message payloads are logged in detail.
+	     To see detailed logging please set default log level to Verbose.
+	     (Component config --> Log output --> Default log verbosity --> Verbose)
```

Comment:
Let's change ChipLoggingDetail to use ESP_LOGD instead of ESP_LOGV and also change the documentation above to Debug. The reason being verbose info can have too much of information which might not be required for debugging?

Reply:
Done